File: Persistencia de objetos/Ejercicio 1/Python/Main.py
from Empleado import Empleado
from ArchivoEmpleado import ArchivoEmpleado

archivo = ArchivoEmpleado("empleados.txt")
archivo.cargarEmpleados()


nuevo = Empleado("María", 29, 3100)
archivo.guardarEmpleado(nuevo)

nuevo = Empleado("Jorge", 26, 5000)
archivo.guardarEmpleado(nuevo)

nuevo = Empleado("David", 21, 4200)
archivo.guardarEmpleado(nuevo)

e = archivo.buscaEmpleado("Julio")
print("Encontrado:", e)

mayor = archivo.mayorSalario(2500)
print("Empleado con mayor salario:", mayor)

File: Persistencia de objetos/Ejercicio 1/Java/Main.java
import java.io.IOException;

public class Main {
    public static void main(String[] args) throws IOException {

        ArchivoEmpleado archivo = new ArchivoEmpleado("Empleados/empleados");

        archivo.guardarEmpleado(new Empleado("Ana", 30, 1000));
        archivo.guardarEmpleado(new Empleado("Luis", 40, 1500));
        archivo.guardarEmpleado(new Empleado("Eva", 25, 800));

        Empleado buscado = archivo.buscaEmpleado("Eva");
        System.out.println("BUSCADO: " + buscado);

        Empleado mayorSal = archivo.mayorSalario(900f);
        System.out.println("MAYOR SALARIO > 900: " + mayorSal);
    }
}
